lib/src/player_view.dart

```diff
@@ -169,3 +192,44 @@ class PlayerViewState extends State<PlayerView> {
           );
   }
 }
+
+class _PictureInPictureAPI implements PictureInPictureAPI {
```

Comment:
Let's rename this according to our naming conventions:
```suggestion
class _PictureInPictureApi implements PictureInPictureApi {
```

Reply:
Address in f297409a932cd2677b095d9f19dbc75f9733a8f4